pkg/api/types_test.go

```diff
@@ -134,3 +134,61 @@ func TestOverlay(t *testing.T) {
 		})
 	}
 }
+
+func TestBuildsImage(t *testing.T) {
+	for _, tc := range []struct {
+		name string
+		conf ReleaseBuildConfiguration
+		want bool
+	}{{
+		name: "not in `images`",
+		conf: ReleaseBuildConfiguration{
+			Images: []ProjectDirectoryImageBuildStepConfiguration{
+				{To: "egami"},
+			},
+		},
+	}, {
+		name: "in `images`",
+		conf: ReleaseBuildConfiguration{
+			Images: []ProjectDirectoryImageBuildStepConfiguration{
+				{To: "image"},
+			},
+		},
+		want: true,
+	}} {
+		t.Run(tc.name, func(t *testing.T) {
+			if ret := tc.conf.BuildsImage("image"); ret != tc.want {
+				t.Errorf("got %v, want %v", ret, tc.want)
+			}
+		})
+	}
+}
+
+func TestIsPipelineImage(t *testing.T) {
+	conf := ReleaseBuildConfiguration{
+		InputConfiguration: InputConfiguration{
+			BaseImages: map[string]ImageStreamTagReference{
+				"base-img": {},
+			},
+			BaseRPMImages: map[string]ImageStreamTagReference{
+				"base-rpm-img": {},
+			},
+		},
+		BinaryBuildCommands:     "make",
+		TestBinaryBuildCommands: "make test-bin",
+		RpmBuildCommands:        "make rpms",
+		Images:                  []ProjectDirectoryImageBuildStepConfiguration{{To: "img"}},
+	}
+	for _, i := range []string{
```

Comment:
Personally I would prefer this to be `t.Run()` based so it's executed in parallel and does not stop on first failure.